Task 16 create css

Sasho878748 opened the pull request.

Sasho878748 committed:
create mob css+html

Sasho878748 committed:
Task-16 final

Yurii-Ilkevych reviewed (commented):
вважаю на декоративному фото яке потім натягнеться на хедер має бути лінійний градієнт linear-gradient(180deg, #000000 0%, rgba(0, 0, 0, 0.55) 54.17%, rgba(0, 0, 0, 0) 100%); колір кнопки має бути background: rgba(0, 0, 0, 0.4);
 трошкі підправити і беде добре

Sasho878748 committed:
Task-16 до опрацювання 1

Додав лінійний градієнт linear-gradient(180deg, #000000 0%, rgba(0, 0, 0, 0.55) 54.17%, rgba(0, 0, 0, 0) 100%) та колір кнопки background: rgba(0, 0, 0, 0.4);

Yurii-Ilkevych committed:
Update task-sixteen.css

Yurii-Ilkevych committed:
Merge branch 'Task-16-create-CSS' of https://github.com/Yurii-Ilkevych/blended1-proj-minimo into Task-16-create-CSS
